app/webpacker/components/select_users_from_wizard.js

```diff
@@ -0,0 +1,35 @@
+class SelectUsersFromWizard {
+
+  constructor() {
```

Comment:
I think you should scope this constructor with a first call that short circuits if not on the right page (RDV wizard Step 2)